shark_turbine/kernel/wave/expansion.py

```diff
@@ -430,7 +434,9 @@ def get_dim_scaling(
                 raise ValueError(
                     "Tile size must be divisible by wave count and vector size"
                 )
-            dim_scaling[constraint.dim] = tile_size // wave_count // vector_size
+            dim_scaling[constraint.dim] = (
+                tile_size // wave_count // num_threads // vector_size
```

Comment:
I dont think the expansion needs to depend on the number of threads. Why do you think so?

Reply:
Also resolved offline. Turns out we do not need this. This was from my misunderstanding that vector_shape == multi-dim threads_per_elem. Where in reality vector_shape is the size of vector owned by a single warp.